src/pages/test-report/cat-be/components/reverse-diagram-modal/reverse-diagram-modal.ts

```diff
@@ -11,8 +11,10 @@ import { getVehicleDetails } from '../../../../../modules/tests/vehicle-details/
 import {
   getVehicleLength,
   getVehicleWidth,
-} from '../../../../../modules/tests/vehicle-details/vehicle-details.cat-be.selector';
+} from '../../../../../modules/tests/vehicle-details/vehicle-details.cat-c.selector';
```

Comment:
Is this in the right file location? I notice that the selector is for cat-c but the file is in the cat-be directory.

Reply:
changed it